### PR TITLE
Added link to public Slack channel in README

### PR DESCRIPTION
**Pull request by dominic-mulligan-arm**

### CODE REVIEW

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 452e481d04c434e631b70d8177dac1010de26494
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=d3533295-b77b-47ce-ae12-deeeb2386047%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

File: README.markdown [hunk 1]
```diff
@@ -27,6 +27,10 @@ The Veracruz team welcome new collaborators and contributions!
 We maintain a list of open issues, ideas for new features, and possible improvements that can be made to Veracruz in our issue tracker, but also welcome ideas for new features and improvements from contributors, too.
 Many issues in our issue tracker are marked as being suitable for new contributors to the project.
 
+Veracruz maintains a public Slack channel for discussion about the project under the Confidential Compute Consortium's workspace.
+You can access this channel [here](https://join.slack.com/t/confidentialcomputing/shared_invite/zt-nckyewk3-WoMUPIrPdxXCCXrjCnFApw).
+Anybody and everybody is welcome to join to meet the team and discuss the project!
```
Comment (ShaleXIONG) [inline README.markdown:32]: Is it a common expression by 'Anybody and everybody'?

**Review by ShaleXIONG — APPROVED** [review]

File: README.markdown (hunk 1 above)
Comment (dominic-mulligan-arm) [reply README.markdown:32]: Yep!

**Review by gbryant-arm — APPROVED** [review]

**Comment by veracruz-project-owner** [comment]
### AWS CodeBuild CI Report

* CodeBuild project: Veracruz
* Commit ID: 05a1be3552b02341ffb960f7eada4f4d7ada30d2
* Result: SUCCEEDED
* [Build Logs](https://t4r5bdzqe9.execute-api.eu-west-2.amazonaws.com/Prod/buildlogs?key=2f10f1fd-0e48-4897-a433-d280c8b1b96f%2Fbuild.log) (available for 30 days)

*Powered by [github-codebuild-logs](https://serverlessrepo.aws.amazon.com/applications/arn:aws:serverlessrepo:us-east-1:277187709615:applications~github-codebuild-logs), available on the [AWS Serverless Application Repository](https://aws.amazon.com/serverless/serverlessrepo/)*

**Comment by dominic-mulligan-arm** [comment]
+1+1 = +1.

**Pull request merged**